setup.py

```diff
@@ -20,7 +20,7 @@
 
 
 min_python_version = "3.10"
-max_python_version = "3.13"  # exclusive
+max_python_version = "3.14"  # exclusive
```

Comment:
This reminds me, the trove classifier needs updating along with the version support table.

Reply:
Updated in 9c59e31a1d8313cad0d3597eb351386c7f7ac9a1

Reply:
Thanks.